fapolicy_analyzer/ui/main_window.py

```diff
@@ -197,7 +197,7 @@ def __pack_main_content(self, page: UIPage):
                 page.dispose()
                 return
 
-        if self.__page:
+        if self.__page is not None:
```

Comment:
Cause of split view when navigating away from rule view after Events was mixed in.

The type after Event mixin must not have had a __bool__ ability.  It also didnt have a string rep, perhaps that was one in the same with the bool rep.

Checking for `not None` here is appropriate regardless, since there is not a valid bool rep of a page.